x-pack/plugins/security_solution/public/common/hooks/eql/helpers.ts

```diff
@@ -33,37 +32,16 @@ export const calculateBucketForDay = (eventTimestamp: number, relativeNow: numbe
   return Math.ceil(minutes / 60);
 };
 
-export const constructWarnings = (timestampIssue: boolean, hits: number, range: Unit): string[] => {
```

Comment:
There's an existing bug with warnings, working on the fix in a separate PR to keep these small-ish.